index.js

```diff
@@ -129,9 +129,9 @@ module.exports = (api, options) => {
     return runner;
   });
 
-  async function runServer(runServer = false) {
+  async function runServer(runServer = false, args = {}, rawArgs = []) {
```

Comment:
just not sure what to do with args it contains headless and serve vue-cli ignores them but not sure becouse I see you removed them from rawArgs